File: Java/1080.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
       Scanner entrada = new Scanner(System.in);

        int n, maior = 0, posicao = 0;

        for(int i = 0; i < 100; i++) {
            n = entrada.nextInt();
            if(n > maior) {
                maior = n;
                posicao = i + 1;
            }
        }

        System.out.println(maior);
        System.out.println(posicao);
    }
}

File: C-C99/1080.c
#include <stdio.h>
int main()
{
    int entrada, i = 0, maior = -1, p;
    while (i < 101)
    {

        scanf("%d", &entrada);
        if (entrada > maior)
        {
            maior = entrada;
            p = i + 1;
        }
        i++;
    }
    printf("%d\n%d\n", maior, p);

    return 0;
}
